### ISSUE TITLE
`jspm normalize <pkg>` fails with 0.17.0-beta.22 on Ubuntu 16.04 x64

### ISSUE THREAD

**Issue by skorlir** [jspm/jspm-cli#1943]
Node:  6.2.0
JSPM: 0.17.0-beta.22
OS:     Ubuntu 16.04
Arch:   x64

------

Here are the steps to reproduce:

```
$ jspm i moment
...
$ jspm normalize moment
```

The following error is logged to stdout:

```
err  TypeError: Cannot read property 'lastIndexOf' of undefined
    at SystemJSNodeLoader.booleanConditional (/media/Data/dev/work/encurate/cognito-proof-of-concept/node_modules/systemjs/dist/system.src.js:4733:28)
    at SystemJSNodeLoader.normalize (/media/Data/dev/work/encurate/cognito-proof-of-concept/node_modules/systemjs/dist/system.src.js:4758:33)
    at Object.exports.normalize (/media/Data/dev/work/encurate/cognito-proof-of-concept/node_modules/jspm/lib/core.js:206:17)
    at /media/Data/dev/work/encurate/cognito-proof-of-concept/node_modules/jspm/cli.js:386:12
    at Object.<anonymous> (/media/Data/dev/work/encurate/cognito-proof-of-concept/node_modules/jspm/cli.js:599:3)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/jspm/jspm.js:33:5)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/jspm/node_modules/liftoff/index.js:197:12)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/jspm/node_modules/liftoff/index.js:164:7)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
```

**Comment by guybedford** [comment]
Thanks for posting. This was fixed in https://github.com/jspm/jspm-cli/commit/416820aadbd5406a2aa83141bac375b034b9a8e8 pending release.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.23.
